docs/whats-new.md

```diff
@@ -1,11 +1,16 @@
-4. **Reusable executors** - [Executors](executors.md) define the environment and other settings for executing jobs. They allow you to reuse configuration of your execution environment across jobs.
-5. **Conditional steps in jobs** - The new `when` clause under steps allows you to run [conditionally run steps](conditional-steps.md) only when certain parameter values are true.
-6. **Orbs** - Orbs are packages of CircleCI configuration that can be shared across projects. Orbs allow you to make a single bundle of jobs, commands, and executors that can reference each other and can be imported into a CircleCI build configuration and invoked in their own namespace. Orbs are registered with CircleCI, with revisions expressed using the [semver](https://semver.org/) pattern.
+# What's new with build processing? 
+
+1. **Build processing** - We are refactoring our build ingestion system to pre-process configuration before running workflows or jobs. Centralizing configuration processing will allow us to provide much better and much earlier feedback on problems with the configuration itself, such as syntax errors or malformed semantics. This will also allow us to add various configuration "sugaring" techniques and make reusable, parameterizable configuration a reality. To use 2.1 configuration you must turn on the Build Processing option under the Advanced tab in your project settings. If you look under the "Configuration" tab on a job detail page you will see both the configuration originally used and the configuration after it has been processed.
+1. **All jobs run in workflows** - With the new build processing system if you run an individual job we will automatically wrap that job in a workflow. We are doing this to create a more consistent experience in both our UI and in the underlying data and execution paths for jobs coming through the system. There are some subtle changes in the UI as a result, the most striking of which is that rather than rerunning an individual job you will have the option to rerun the workflow that originally ran the job. For single-job builds this change will alter what you see when you choose to rebuild, but it should be functionally equivalent to the previous behaviors.
+
+# What's in in 2.1 configuration?
+1. **Workflows now share the config version with the rest of the config** - You no longer need a separate version under the `workflows` stanza in configuration because from 2.1 onward we will version all of configuration together rather than versioning individual stanzas.
+1. **Deprecation of some previous beta syntax** - for the relatively few people who used very early preview versions of CircleCI 2.0 there were some different names for certain built-in commands. In some cases we have deprecated these (TODO: get the specific list) and/or are showing warnings about them.
```

Comment:
This wasn’t in the beta - it was “pre alpha” syntax that we shared with some early partners. 

We have not deprecated it, we have removed it. 

Some of that syntax was inadvertently listed in some config examples in our docs after launch, so a very small number of customers might have it in their config.

Reply:
Nit: `for` should be capitalized to maintain consistency

Reply:
changing language to "alpha" and capitalizing For